etc/scripts/licenses/report_license_rules.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import io
+
+import click
+import csv
+
+from commoncode.cliutils import PluggableCommandLineOption
+from licensedcode.models import load_licenses
+from licensedcode.models import load_rules
+
+
+LICENSES_FIELDNAMES = [
+    'key', 'short_name', 'name', 'category', 'owner',  'text', 'words_count', 'notes',
+    'minimum_coverage', 'homepage_url', 'is_exception', 'language', 'is_unknown',
+    'spdx_license_key', 'text_urls', 'other_urls', 'standard_notice',
+    'license_filename', 'faq_url', 'ignorable_authors',
+    'ignorable_copyrights', 'ignorable_holders', 'ignorable_urls', 'ignorable_emails',
+    'osi_license_key', 'osi_url', 'other_spdx_license_keys',
+]
+
+
+RULES_FIELDNAMES = [
+    'identifier', 'license_expression', 'relevance', 'text', 'words_count', 'category',
+    'is_false_positive', 'is_license_text', 'is_license_notice',  'is_license_tag',
+    'is_license_reference', 'is_license_intro', 'has_unknown', 'only_known_words',
+    'notes', 'referenced_filenames', 'minimum_coverage', 'ignorable_copyrights',
+    'ignorable_holders', 'ignorable_authors', 'ignorable_urls', 'ignorable_emails',   
+]
+
+
+def write_data_to_csv(data, output_csv, fieldnames):
+    
+    with open(output_csv,'w',encoding='utf-8-sig',newline='') as f:
```

Comment:
`,encoding='utf-8-sig',newline=''` are suspicious to me... IMHO you are best of without them